src/robusta/core/playbooks/internal/discovery_events.py

```diff
@@ -20,17 +20,18 @@
 @action
 def cluster_discovery_updates(event: KubernetesAnyChangeEvent):
```

Comment:
Can we rely on two subsequent calls to cluster_discovery_updates executing in order on the same resource?

Reply:
I think this is the case in most cases, but in theory, the order might change
I don't think k8s resource changes are that frequent (on busy clusters we get something like 10/sec on all the cluster)